mayastor-test/test_snapshot.js

```diff
@@ -196,6 +214,32 @@ describe('snapshot', function () {
       assert.equal(res.uuid.startsWith(replicaUuid + '-snap-'), true);
       assert.equal(res.share, 'REPLICA_NONE');
       assert.match(res.uri, /^bdev:\/\/\//);
+      // Wait 1 second so that the 2nd snapshot has a different name and can
+      // created successfully
```

Comment:
typo: "...and can **be** created successfully"

Reply:
Good spot, fixed.